template/function/serverless.yml

```diff
@@ -1,21 +1,29 @@
-service: app
+service: TPL_SERVICE_NAME
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
+  # general settings
+  name: aws
+  stage: dev
+  region: TPL_REGION
+  profile: default
+  deploymentMethod: direct
+  # global function settings (apply to each declared function)
+  runtime: provided.al2
+  memorySize: 1024
+  timeout: 6
+  logRetentionInDays: forever
+  layers:
+    - ${bref:layer.php-TPL_PHP_VERSION}
```

Comment:
I'm afraid of putting the layer on all functions, if a user wants to use FPM later it will mess things up.